content/docs/command-reference/import.md

```diff
@@ -276,3 +278,133 @@ outs:
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
+
+## Example: Chained imports
+
+DVC supports importing data importing data that was imported into the source
+repo, as long as all repos in the import chain are accessible from the final
+destination repo. Each source repo's default DVC remote must also be accessible
+from the final destination repo.
+
+Consider an example where we start with two DVC repos, A and B, with the
+following contents:
+
+```
+/repo/a
+├── foo
+└── foo.dvc
+
+/repo/b
+├── bar
+└── bar.dvc
+```
+
+Now, we create a third DVC repo, C, with the following contents:
+
+```
+/repo/c
+└── dir
+    ├── bar
+    ├── bar.dvc
+    ├── foo
+    ├── foo.dvc
+    ├── subdir
+    │   └── baz
+    └── subdir.dvc
+```
+
+Repo C, contains a data directory tracked with `dvc add` (`dir/subdir`), as well
+as two separate imports (`dir/foo` and `dir/bar`). The imports were created via:
+
+```dvc
+# In repo C
+$ dvc import /repo/a foo -o dir/foo
+$ dvc import /repo/b bar -o dir/bar
+```
+
+If we examine the contents of `dir/foo.dvc`, we can see that it is imported from
+repo A:
+
+```yaml
+# dir/foo.dvc
+md5: d652071a8f0fd9f5c74d9348a468dec5
+frozen: true
+deps:
+- path: foo
+  repo:
+    url: /repo/a
+    rev_lock: 32ab3ddc8a0b5cbf7ed8cb252f93915a34b130eb
+outs:
+- md5: acbd18db4cc2f85cedef654fccc4a4d8
+  size: 3
+  path: foo
+outs:
+- md5: 630bd47b538d2a513c7d267d07e0bc44.dir
+  size: 3
+  nfiles: 1
+  path: subdir
```

Comment:
This doesn't belong in `dir/foo.dvc` of your example I think right @pmrowla ? I'll remove it for now.

Reply:
```
outs:
- md5: acbd18db4cc2f85cedef654fccc4a4d8
  size: 3
  path: foo
```
this part was part of a bad copy/paste and should be removed. The `.dir` hash is supposed to be there.

`dir/` contains `foo` which is imported, and a `subdir/` which is a new directory for this repo, tracked by `dvc add dir/subdir`

The point is to show that later in the chain, we can import `dir/` which contains both dvc tracked data specific to this repo, as well as imports from other DVC repos

Reply:
A bit confused. TBH I didn't have a chance to actually test all this. Mind taking a look at my rewrite and confirm it's correct? Thanks